lib/react_client/js_interop_helpers.dart

```diff
@@ -36,12 +53,20 @@ final _GetPropertyFn getProperty = (() {
 ///
 /// Necessary because `external operator[]=` isn't allowed on JS interop classes
 /// (see: https://github.com/dart-lang/sdk/issues/25053).
+///
+/// __Defined in this package's React JS files.__
 final _SetPropertyFn setProperty = (() {
-  _eval(r'''
-    function _setProperty(obj, key, value) {
-      return obj[key] = value;
-    }
-  ''');
+  try {
+    // If this throws, then the JS function isn't available.
+    _getProperty(new EmptyObject(), null);
+  } catch(_) {
+    throw new _MissingJsMemberError('_setProperty',
+        'Be sure to include React JS files included in this package '
+        '(which has this and other JS interop helper functions included) '
+        'or, alternatively, define the function yourself:\n'
+        '    function _setProperty(obj, key, value) { return obj[key] = value; }'
```

Comment:
Fixed in https://github.com/cleandart/react-dart/pull/90/commits/004982773768b035a14427b55397fecd36ba672d